### PR TITLE
Multi Bug - L10N Screenshot Improvements

### PR DESCRIPTION
**Pull request by st3fan**
Multi Bug - L10N Screenshot Improvements

### CODE REVIEW

File: L10nSnapshotTests/L10nSnapshotTests.swift [hunk 1]
```diff
@@ -182,4 +182,77 @@ class L10nSnapshotTests: L10nBaseSnapshotTests {
...
+        app.tables["History List"].cells.element(boundBy: 2).press(forDuration: 2.0)
+        snapshot("19HistoryTableContextMenu-01")
+    }
+
+    func test20BookmarksTableContextMenu() {
+        let app = XCUIApplication()
+        loadWebPage(url: "http://wopr.norad.org/~sarentz/fxios/testpages/index.html", waitForOtherElementWithAriaLabel: "body")
+        app.buttons["TabToolbar.menuButton"].tap()
+        app.cells["AddBookmarkMenuItem"].tap()
+        sleep(2) // There must be a nicer way to wait for the menu to go away
```
Comment (npark-mozilla) [inline L10nSnapshotTests/L10nSnapshotTests.swift]: you could use waitforNoExistence() in XCUITests/Utils/BaseTestCase.swift

File: L10nSnapshotTests/L10nSnapshotTests.swift [hunk 2]
```diff
@@ -182,4 +182,77 @@ class L10nSnapshotTests: L10nBaseSnapshotTests {
...
+        app.tables["ReadingTable"].cells.element(boundBy: 0).swipeLeft()
+        snapshot("23ReadingListTableRowMenu-02")
+    }
+
+    func test24BookmarksListTableRowMenu() {
+        let app = XCUIApplication()
+        loadWebPage(url: "http://wopr.norad.org/~sarentz/fxios/testpages/index.html", waitForOtherElementWithAriaLabel: "body")
+        app.buttons["TabToolbar.menuButton"].tap()
+        app.cells["AddBookmarkMenuItem"].tap()
+        sleep(2) // There must be a nicer way to wait for the menu to go away
```
Comment (npark-mozilla) [inline L10nSnapshotTests/L10nSnapshotTests.swift]: same here, waitForNoExistence() could be used

**Comment by garvankeeley** [comment]
I see no screenshot for the top-level settings screen, and I don't see it added in this bug

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
SnapshotHelper.swift | 1 | Files should have consistent header comments. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by st3fan** [comment]
I cherry-picked the individual commits up to _Bug 1393634 - Update SnapshotHelper.swift_.

**Pull request closed without merging**